### ISSUE TITLE
Is there any plan to implement transformer-xl?

### ISSUE THREAD

**Issue by GlassyWing** [kpot/keras-transformer#7]

**Comment by kpot** [comment]
Yes, I do have plans to implement Transformer-XL, but cannot promise how soon.

**Comment by GlassyWing** [comment]
Wow, you're amazing. I'm looking forward to it

**Comment by sergree** [comment]
I'm looking forward to it too. Thank you!

**Comment by alsm168** [comment]
I'm looking forward to it too. Thank you!

**Comment by zbloss** [comment]
Hey Kpot, has there been any movement on this? I'm looking to do something similar and want to make sure we are duplicating work.
